cime_config/tests.py

```diff
@@ -239,6 +239,7 @@
         "tests" : (
             "ERS_P480_Ld5.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF.mpaso-jra_1958",
             "PEM_P480_Ld5.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF.mpaso-jra_1958",
+            "SMS_P480_Ld5.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF-TMIX.mpaso-jra_1958",
```

Comment:
@jonbob, does this look right?

I was able to run this on Chrysalis by excluding `mpaso-jra_1958` but didn't know how to test that part.

Reply:
I can do that when I work on this PR -- there is a bit of a trick to specifying tests that use testdefs

Reply:
Okay, thanks!

Reply:
The easiest way is to add machine_compiler to the test name.
./creat_test SMS_P480_Ld5.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF-TMIX.chrysalis_intel.mpaso-jra_1958

Reply:
Thanks @rljacob!